migrations/20230714125055-create_stories_table.sql

```diff
@@ -0,0 +1,14 @@
+-- +migrate Up
+
+CREATE TABLE IF NOT EXISTS stories (
+    id SERIAL PRIMARY KEY,
+    author_id INT REFERENCES users(id),
+    story_content TEXT,
```

Comment:
Since this column already belongs to the story's table, we can just name it "content".

Think of each column as the table records' property. E.g.:

* A story's ID
* A story's author ID
* A story's story content &rarr; it makes sense to remove the redundant word.
* A story's "created at" time
* A story's ...